meson.build

```diff
@@ -214,6 +214,7 @@ if cairo_dep.found()
   else
     check_funcs += [['cairo_user_font_face_set_render_color_glyph_func', {'deps': cairo_dep}]]
   endif
+  check_funcs += [['cairo_font_options_get_custom_palette_color', {'deps': cairo_dep}]]
```

Comment:
This should go inside the else branch above, as we can't use this when we are building subproject.

Reply:
I'll move there.  Need to roll forward the subproject dependency when the API lands in cairo.

Reply:
Indeed, once it lands we can do that.

Reply:
It landed